cadasta/search/views/async.py

```
@@ -29,18 +29,29 @@ class Search(APIPermissionRequiredMixin, ProjectMixin, APIView):
     def get_perms_objects(self):
         return [self.get_project()]
 
-    def get(self, request, *args, **kwargs):
-        query = request.query_params.get('q')
+    def post(self, request, *args, **kwargs):
+        query = request.data.get('q')
+        start_idx = self.convert_field_to_int(request.data, 'start', 0)
```

I wasn't questioning the need for casting to `int`. I was wondering whether it's necessary to implement this into a method if it might be possible to make this explicit in this line. I notice that you catch `ValueError` exceptions in the method; I guess that was the reason for putting it into a method. Is it likely that a value that cannot be cast to `int` is provided to any of these parameters (`draw`, `start`, `length`)? If it's likely under which circumstances will this happen? If it's not likely, it might be better not to catch the exception and fail loudly; otherwise there's a risk of introducing a bug that will be difficult to work out.